olga.yepikhina/lecture 1/site.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Olga Y Lecture 1 </title>
+    <style>
+        table,
+        tr,
+        th,
+        td {
+            border: 2px solid olivedrab;
+            padding: 1px;
+        }
+
+        .theme {
+            color: mediumvioletred;
+        }
+
+        .orderlist {
+            color: maroon;
+            font-size: 18px;
+        }
+    </style>
+</head>
+
+<body>
+    <textarea rows="1" cols="50">
+    Welcome on my HTML-Pokemon website :D
+                </textarea>
```

Comment:
textarea tag is not properly aligned :) 
I also suggest using 2 spaces intent, not 4 spaces. (kinda standard in the web dev)
Give me a favor, add .editorconfig file to the root of the project with content like. It will make 2 spaces indent standard for all of us in the repo :) 
```
# editorconfig.org

root = true

[*]
charset = utf-8
indent_size = 2
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true

[*.md]
trim_trailing_whitespace = false
```